Description: Increase the timeout for processing the ANTLR grammar. This fixes a build failure on slow machines.
Author: Emmanuel Bourg <ebourg@apache.org>
Forwarded: no
--- a/buildSrc/build.gradle
+++ b/buildSrc/build.gradle
@@ -115,7 +115,7 @@
     def grammars = fileTree(src).include("**/*.g")
     main = "org.antlr.Tool"
     classpath = configurations.antlr3
-    args = ["-o", dest, grammars.files].flatten()
+    args = ["-Xconversiontimeout", "30000", "-o", dest, grammars.files].flatten()
     // See RT-30955. This should be removed when JDK-8015656 is fixed
     ignoreExitValue = true
 }
